python.conf

```
@@ -0,0 +1,20 @@
+[handlers]
+keys=HTTPSHandler
+
+[handler_HTTPSHandler]
+class=loggly.handlers.HTTPSHandler
+formatter=jsonFormat
+args=('https://logs-01.loggly.com/inputs/23df0669-8b3d-4c0a-a1be-ea9ef7cd44d9/tag/python','POST')
```

We shouldn't need this handler if we're relying on the set up defined in your `app.py`, but importantly we definitely don't want the token here!
```suggestion
args=('https://logs-01.loggly.com/inputs/<token>/tag/python','POST')
```
(Don't forget to now rotate the token!)

---

loggly token has been rotated.